base32h.js

```diff
@@ -120,16 +120,20 @@
         return b[0]*2**32 + b[1]*2**24 + b[2]*2**16 + b[3]*2**8 + b[4];
     }
 
-    // This is cursed
-    function pad(i,w,z) {
-        z = z || '0';
-        w = w || 8;
-        i += '';
-        var o = i.length % w;
-        if (o) {
-            return [...Array(w - o).fill(z), ...i].join('');
+    /**
+     * Pad a string to a given alignment.
+     * @param {string} value - The string to pad.
+     * @param {number} alignment - The alignment size.
+     * @param {string} char - The character to pad with.
+     * @returns {string} - The padded string.
+     */
+    function pad(value, alignment = 8, char = '0') {
```

Comment:
Default parameters are [unsupported in all versions of Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#browser_compatibility); as much as I hate IE, too, I'd like to maintain compatibility with it if possible.